### DISCUSSION TITLE
Lesson 6: forge is not installing

### DISCUSSION THREAD

**Discussion by AnouarBF** [Cyfrin/foundry-full-course-cu#47] [Q&A] [answered]
foundry runs in my terminal well. But the issue is it doesn't download forge. it only loads cast anvil and chisel but not forge!
Here is what shows : 
```
foundryup: installed - forge 0.2.0 (08a629a 2023-06-03T00:04:22.625130135Z)
foundryup: warning: 
There are multiple binaries with the name 'forge' present in your 'PATH'.
This may be the result of installing 'forge' using another method,
like Cargo or other package managers.
You may need to run 'rm /usr/bin/forge' or move '/home/anouar/.foundry/bin'
in your 'PATH' to allow the newly installed version to take precedence!

foundryup: installed - cast 0.2.0 (08a629a 2023-06-03T00:04:22.625130135Z)
foundryup: installed - anvil 0.1.0 (08a629a 2023-06-03T00:05:11.227787246Z)
foundryup: installed - chisel 0.1.1 (08a629a 2023-06-03T00:05:09.099075732Z)
foundryup: done!
```

**Comment by cromewar** [comment]
Hello @AnouarBF which command did you use to install foundry, did you used this one?
`curl -L https://foundry.paradigm.xyz | bash`

If so, you can use the command on the output `rm /usr/bin/forge` and try installing again.

**Reply by AnouarBF** [reply]
Yeah I used it and followed the instructions shown on the terminal. But that doesn't work. I tried to ask ChatGPT and PHIND but found nothing ??

**Reply by AnouarBF** [reply]
I also tried to install foundry on another directory but get the same thing

**Comment by AnouarBF** [comment, marked as answer]
Finally solved! the command instructions shows that you should try ```rm /usr/bin/forge``` 
but the one that worked for me is ```sudo rm /bin/forge``` then I closed the terminal, reopen it again and worked !

**Comment by ghost** [comment]
Thank you very much @AnouarBF this helped me a lot
